### ISSUE TITLE
mock -bootstrap-chroot doen't work on RHEL7

### ISSUE THREAD

**Issue by mkutlak** [rpm-software-management/mock#74]
Steps to reproduce:
```
$ mock --init --configdir ...
```

Ends with:
```
ERROR: Command failed:
/usr/bin/systemd-nspawn -q -M 5832c55c5cee4553aae09e32ce2bab46 -D /var/lib/mock/13371337-bootstrap/root 
--setenv=LANG=en_US.UTF-8 --setenv=TERM=vt100 --setenv=SHELL=/bin/bash --setenv=LC_MESSAGES=C 
--setenv=HOSTNAME=mock --setenv=PROMPT_COMMAND=printf "\033]0;<mock-chroot>\007" --setenv=HOME=/builddir 
--setenv=PATH=/usr/bin:/bin:/usr/sbin:/sbin --setenv=PS1=<mock-chroot> \s-\v\$  /usr/bin/yum 
--installroot /var/lib/mock/13371337/root/ --skip-broken install abrt-addon-ccpp shadow-utils gdb rpm


Directory /var/lib/mock/13371337-bootstrap/root lacks the binary to execute or doesn't look 
like a binary tree. Refusing.
```

Package versions:
```
$ rpm -q mock systemd
mock-1.4.1-1.el7.noarch
systemd-219-39.el7.x86_64
```

**Comment by mkutlak** [comment]
```
$ mock --init --configdir . --resultdir log
```

```
config_opts['root'] = '13371337'
config_opts['target_arch'] = 'x86_64'
config_opts['chroot_setup_cmd'] = '--skip-broken install abrt-addon-ccpp shadow-utils gdb rpm'
config_opts['plugin_conf']['ccache_enable'] = False
config_opts['plugin_conf']['yum_cache_enable'] = False
config_opts['plugin_conf']['root_cache_enable'] = False
config_opts['plugin_conf']['bind_mount_enable'] = True
config_opts['plugin_conf']['bind_mount_opts'] = { 'create_dirs': True, 'dirs': [] }
config_opts['plugin_conf']['bind_mount_opts']['dirs'].append(('/var/cache/retrace-server/fedora-26-x86_64', '/var/cache/retrace-server/fedora-26-x86_64'))


config_opts['yum.conf'] = """
[main]
cachedir=/var/cache/yum
debuglevel=1
reposdir=/dev/null
logfile=/var/log/yum.log
retries=20
obsoletes=1
gpgcheck=0
assumeyes=1
syslog_ident=mock
syslog_device=

#repos

[fedora]
name=fedora-26-x86_64
baseurl=file:///var/cache/retrace-server/fedora-26-x86_64/
failovermethod=priority
"""
```

**Comment by ellert** [comment]
This should be fixed with the proposed changes in #141.

**Comment by xsuchy** [comment]
Closing then.
